chantal-sockets.rb

```diff
@@ -0,0 +1,66 @@
+rides = [
+  {driver_id: 'DR0004', date: '3rd Feb 2016', cost: 5 , rider_id: 'RD0022', rating: 5},
+  {driver_id: 'DR0001', date: '3rd Feb 2016', cost: 10, rider_id: 'RD0003', rating: 3},
+  {driver_id: 'DR0002', date: '3rd Feb 2016', cost: 25, rider_id: 'RD0073', rating: 5},
+  {driver_id: 'DR0001', date: '3rd Feb 2016', cost: 30, rider_id: 'RD0015', rating: 4},
+  {driver_id: 'DR0003', date: '4th Feb 2016', cost: 5 , rider_id: 'RD0066', rating: 5},
+  {driver_id: 'DR0004', date: '4th Feb 2016', cost: 10, rider_id: 'RD0022', rating: 4},
+  {driver_id: 'DR0002', date: '4th Feb 2016', cost: 15, rider_id: 'RD0013', rating: 1},
+  {driver_id: 'DR0003', date: '5th Feb 2016', cost: 50, rider_id: 'RD0003', rating: 2},
+  {driver_id: 'DR0002', date: '5th Feb 2016', cost: 35, rider_id: 'RD0066', rating: 3},
+  {driver_id: 'DR0004', date: '5th Feb 2016', cost: 20, rider_id: 'RD0073', rating: 5},
+  {driver_id: 'DR0001', date: '5th Feb 2016', cost: 45, rider_id: 'RD0003', rating: 2}
+]
+
+driver_rides = {}
+
+rides.each do |ride|
+    driver_id = ride[:driver_id]
+    if driver_rides.key?(driver_id)
+        driver_rides[driver_id] += 1
```

Comment:
We may have already talked about this in person, but it looks like your editor is set up to use 4 spaces for indentation. Our style guide at Ada is to use 2, which will be important once we start collaborating on group projects across multiple computers using git. If you haven't already, please change this setting.